PurchasesTests/Resources/receipts/verifyReceipt.txt

```diff
@@ -0,0 +1,369 @@
+{
```

Comment:
I figured it might be useful to have the file around for testing purposes, but it differs enough from the actual receipt that I'll probably replace it with a JSON representation of the actual receipt.